tasks/deploy/config/mainnet/stusd-182day.ts

```diff
@@ -0,0 +1,84 @@
+import { Address, keccak256, parseEther, toBytes, zeroAddress } from "viem";
+import {
+    HyperdriveInstanceConfig,
+    getLinkerDetails,
+    normalizeFee,
+    parseDuration,
+    toBytes32,
+} from "../../lib";
+import {
+    SIX_MONTHS,
+    STUSD_ADDRESS_MAINNET,
+    USDA_ADDRESS_MAINNET,
+} from "../../lib/constants";
+import { MAINNET_ERC4626_COORDINATOR_NAME } from "./erc4626-coordinator";
+import { MAINNET_FACTORY_NAME } from "./factory";
+
+// The name of the pool.
+export const MAINNET_STUSD_182DAY_NAME = "ElementDAO 182 Day stUSD Hyperdrive";
+
+// The initial contribution of the pool.
+const CONTRIBUTION = parseEther("100");
+
+export const MAINNET_STUSD_182DAY: HyperdriveInstanceConfig<"ERC4626"> = {
+    name: MAINNET_STUSD_182DAY_NAME,
+    prefix: "ERC4626",
+    coordinatorAddress: async (hre) =>
+        hre.hyperdriveDeploy.deployments.byName(
+            MAINNET_ERC4626_COORDINATOR_NAME,
+        ).address,
+    deploymentId: keccak256(toBytes(MAINNET_STUSD_182DAY_NAME)),
+    salt: toBytes32("0x69420"),
+    extraData: "0x",
+    contribution: CONTRIBUTION,
+    fixedAPR: parseEther("0.0829"),
+    timestretchAPR: parseEther("0.05"),
```

Comment:
we've seen this go as high as 35%. it would be good to test to see that it can hit that APY with this timestretch. i am pretty sure it can, but it won't be smooth.

Reply:
Gotcha, when I looked on DeFiLlama, I didn't see anything above 10%. I'll update it, but let's follow up with this next week. I must have missed something.